### PR TITLE
Rails-React Setup

### PR DESCRIPTION
**Pull request by jonathastavares**
In this milestone I created a Rails App and connected it through an API endpoint to my React frontend App, the actions are stored in Redux reducers.

Built using:

- Ruby on Rails
- React
- Redux

### CODE REVIEW

File: app/javascript/components/HelloWorld.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    .catch(error => console.log(error));
+  }
+}
+
+export function getThingsSuccess(json) {
+  return {
+    type: GET_THINGS_SUCCESS,
+    json
+  }
+}
```
Comment (elisha2kyakpo1) [inline app/javascript/components/HelloWorld.js]: - [ ] Kindly change this file name to greeting, this is how the project instructions are. See the image below.

greeting

File: app/controllers/v1/things_controller.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+class V1::ThingsController < ApplicationController
+  def index
```
Comment (elisha2kyakpo1) [inline app/controllers/v1/things_controller.rb]: [OPTIONAL]

- [ ] We can rename this controller to something like greetingsController to match the project requirements

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi 👋 @jonathastavares,

You have really done a great job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
- The PR description and title looks good 👍
- No linter errors 👍
- The project works as expected 👍
- The readme file is well documented 👍


## Required Changes ♻️
_Check the comments under the review._
- [ ] Kindly add 5 messages (greetings) to the app, we can achieve this by creating some predefined messages in a seed file. This is part of the project requirement.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Dipeshtwis — APPROVED** [review]
Hi @jonathastavares, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**